{{cookiecutter.app_name}}/.github/workflows/deploy_mock_server.yml

```
@@ -0,0 +1,22 @@
+name: Deploy Mock Server
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  deploy:
+    name: Deploy mock server
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+
+      - name: Deploy to fly
+        run: flyctl deploy --remote-only
+        env:
+          FLY_API_TOKEN: {{ "${{ secrets.FLY_API_TOKEN }}" }}
```

Could we add a tailing blank line here?

```suggestion
          FLY_API_TOKEN: {{ "${{ secrets.FLY_API_TOKEN }}" }}

```

---

@carryall All of the files already have a trailing blank line